src/SPIN/compute_spin.cpp

```diff
@@ -148,10 +148,12 @@ void ComputeSpin::compute_vector()
   int i;
   int countsp, countsptot;
   double mag[4], magtot[4];
+  double m2, m2tot;
+  double m4, m4tot;
```

Comment:
unused variables

Reply:
Hey Richard,
I removed the declaration of those quantities (binder,m2,m2tot,m4,m4tot). I do not use them in compute/spin anymore.